### PR TITLE
Components

### PR DESCRIPTION
**Pull request by paulinagonzalezc**
# Bookstore: initialize project with components

In this project I laid foundations for my Bookstore website. I created the React part of the app and set up routing using React Router.

General requirements

* Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config)
* Make sure that you used correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)
* Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
* Follow our list of the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
* Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

Project requirements

* Initialize React app
* The building blocks of your app should be set as re-usable components. Create a directory for your components.
* Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two Routes and Links for the app's navigation:
  - Books - the default view:
    * Should display the list of books 
    * Should display a form for adding a book (no functionality yet).
  - Categories:
    * Should display a button "Check status" only.
* Styling is not required at this point.

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <Book
+        title={listBooks[1].title}
+        author={listBooks[1].author}
+        category={listBooks[1].category}
+      />
+      <Book
+        title={listBooks[2].title}
+        author={listBooks[2].author}
+        category={listBooks[2].category}
+      />
```
Comment (emmiiorji) [inline src/components/Books.js]: Considering that you might not always know the number of elements in the array and also to make the code DRYer, I suggest that you use a map statement here.

e.g:

```
     {listBooksooks.map((book) => (
            <Book
              title={book.title}
              author={books.author}
              category={book.category}
            />
      ))}
```

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @paulinagonzalezc, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adhered to Gitflow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (paulinagonzalezc) [reply src/components/Books.js]: Hi @emmiiorji , thank you for the tip! I'll go ahead and make the change 😄

**Review by FrederickMih — APPROVED** [review]
## Hi 🙋  @paulinagonzalezc , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlight

- Descriptive PR ✅
- All Linter checks passed ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**